src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+$calendar-day-width: 100px;
+$calendar-day-gap: 1px;
+$the-last-day: 31;
+$feb-day-count: 28;
+$week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
same question for .container

Reply:
You could add mix class to calendar and add postition styles to it. This way we'll not gonna add any extra blocks to our page and more accurately follow the task.